src/what.sh

```diff
@@ -4,6 +4,11 @@
 # For more details, see functions _What_usage and _What_help
 # as well as _What_info.
 
+## COLOR
+GREEN="\033[0;32m"
+NORMAL="\033[0;00m"
+RED="\033[0;31m"
```

Comment:
On source, these will be in the environment and could easily be overwritten. I would obfuscate them like the functions: `_What_GREEN` etc